[All Hosts] (dialog) document error 12011 (#3614)
OfficeDev/office-js-docs-pr: docs/reference/javascript-api-for-office-error-codes.md

|12006|||The dialog box was closed, usually because the user chooses the **X** button. Thrown within the dialog and triggers a `DialogEventReceived` event in the host page.|
|12007|||A dialog box is already opened from this host window. A host window, such as a task pane, can only have one dialog box open at a time. Thrown by call of `displayDialogAsync`.|
|12009|||The user chose to ignore the dialog box. This error can occur in online versions of Office, where users may choose not to allow an add-in to present a dialog. Thrown by call of `displayDialogAsync`.|

> |12011|||The user's browser is configured is configured in a way that blocks popups. This error can occur in online versions of Office if the browser is Safari and it is configured to block popus or the browser is Edge Legacy and the add-in domain is in a different security zone as the domain the dialog is trying to open. Thrown by call of `displayDialogAsync`.|

Reviewer: ```suggestion
|12011|||The user's browser is configured in a way that blocks popups. This error can occur in Office on the web if the browser is Safari and it's configured to block popups or the browser is Edge Legacy and the add-in domain is in a different security zone from the domain the dialog is trying to open. Thrown by call of `displayDialogAsync`.|
```
